SUPPORTED-HARDWARE.md

```diff
@@ -1,25 +1,48 @@
+The Bottlerocket kernel for metal is configured to include drivers for a wider spread of 10G+ Ethernet NICs in their base configuration (no model-specific FPGA offloading support and similar) as well as common RAID controllers.
 
-Bottlerocket's kernel configuration isn't as extensive as some other distros, so it's possible that it is missing drivers for your specific hardware.
-If so, please [submit an issue](https://github.com/bottlerocket-os/bottlerocket/issues/new?assignees=&labels=&template=feature.md) and we'll work on integrating the proper kernel modules for your hardware!
+Beyond that, the amount of drivers included in the Bottlerocket kernels has been reduced substantially comparing to common general purpose Linux distributions.
+The aim is to keep Bottlerocket images as lean as possible, while trying to maintain a good out-of-the-box coverage.
 
-## Confirmed support
+It is possible that Bottlerocket is missing drivers for your specific hardware.
+Please [submit an issue](https://github.com/bottlerocket-os/bottlerocket/issues/new?assignees=&labels=&template=metal_driver.md) to open a discussion on inclusion of additional drivers.
 
-Bottlerocket is tested on and known to work with the hardware below.
+## Limitations of hardware support to be added
+
+Adding drivers that are part of the upstream Linux source tree is an easy fix for certain target platforms.
+However, there are limitations of what to add to the Bottlerocket metal variant in order to accommodate Bottlerocket's core principles of keeping it small and simple.
+If you want to create a custom variant that for example contains specific drivers, the current work towards out-of-tree builds will help you achieve that.
+Work for that is currently underway and can be tracked in [issue #2669](https://github.com/bottlerocket-os/bottlerocket/issues/2669).
+Until out-of-tree builds land the following limitations apply to the available Bottlerocket variants:
+
+* There is no plan to add out-of-tree drivers to Bottlerocket images.
+* There is no plan to add additional CPU architectures.
+* There is no plan to add drivers for embedded devices in the core images.
+
+If you have questions about these limitations or want to debate them, feel free to open an issue or start a discussion.
 
-| Server model | CPU | BIOS/UEFI | Network Card | Disk | RAID/Storage controller |
-| --- | --- | --- | --- | --- | --- |
-| Supermicro SYS-E200-8D | Intel Xeon D-1528 | BIOS & UEFI | Intel i350 1G & 10G | SATA SSD, NVME | N/A |
-| Dell R240 | Intel Xeon E2236 | BIOS & UEFI | Broadcom BCM5720 1G | SATA SSD (RAID0) | PERC H730P |
-| Dell R620 | Intel Xeon E5-2660 | BIOS | Intel i350 1G | SATA HDD | PERC H710P |
-| HP DL20 | Intel Xeon E2234 | BIOS | HPE 361i 1G | SATA SSD | HPE Smart Array S100i |
+## Testing
 
-## Reported Support
+The AWS Bottlerocket team does base level functional testing on a limited set of server configurations they have available (See below).
```

Comment:
'base level' confuses me - I'm not sure if that has a meaning beyond 'simple'. Would 'basic' work better here?

Additionally, "see below" revised to be more specific and should be a hyperlink (hash link to the heading) to the correct section for better clarity.